updates/compatibility/blueprint_shotgun.lua

```diff
@@ -0,0 +1,33 @@
+local tech_lv_0 = data.raw['technology']['blueprint-shotgun']
```

Comment:
this file needs wrapped in `if mods['blueprint-shotgun']` so that it doesn't cause errors when that mod isn't active.

Reply:
dang it that's been in my head like 10-20 times whilst prepping the compatibility, dunno how i forgot 💀